Cargo.toml

```
@@ -47,7 +47,7 @@ tokio-util = { version = "0.3.1", features = ["codec"] }
 tokio = { version = "0.2", features = ["io-util"] }
 bytes = "0.5.2"
 http = "0.2"
-log = "0.4.1"
+tracing = "0.1"
```

We should probably enable the `log` feature for downstream users who expect `log` records to be emitted.
```suggestion
tracing = { version = "0.1", default-features = false, features = ["std", "log"] }
```